src/delete.js

```diff
@@ -1,5 +1,5 @@
-import indexUpdate from './updateIndex.js';
-import getFromStorage, { saveInStorage } from './storage.js';
+import indexUpdate from './module/updateIndex.js';
+import getFromStorage, { saveInStorage } from './module/storage.js';
```

Comment:
- [ ] Also try to mock for functions that used localStorage and show that localstorage is working fine after adding or deleting the task.